### PR TITLE
Add Stylelint and rc file

### LINKED ISSUE #30: Add style linting
**Is your feature request related to a problem? Please describe.**
We currently don't have style linting for our code.

**Describe the solution you'd like**
We should have an npm script `test:lint:css`

These should be installed as dev dependencies 
```
stylelint
stylelint-config-recommended
stylelint-config-styled-components
stylelint-processor-styled-components
```
We should add this file:
`.stylelintrc`
```
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-recommended",
    "stylelint-config-styled-components"
  ]
}
```

We should fix any style lint issues.
We should update the npm script `test` to include the style lint task.

**Describe alternatives you've considered**
N/A

**Testing notes**
When we run `npm run test:lint:css`, the style linting should work. 
When we run `npm run test` it should run the style lints as well as the eslint & unit tests. 

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #30

Adds style linting

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by bcmn — DISMISSED** [review]

**Review by bcmn — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -11,8 +11,9 @@
     "storybook": "NODE_ENV=development start-storybook -p 8080 -c .storybook",
     "storybook:export": "build-storybook -c .storybook -o public",
     "speculate": "speculate",
-    "test": "npm run test:lint && npm run test:unit",
+    "test": "npm run test:lint && test:lint:css && npm run test:unit",
```
Comment (sareh) [inline package.json]: This needs to be `npm run test:lint:css`. Currently this doesn't run.

**Review by sareh — CHANGES_REQUESTED** [review]
The stylelint work looks good. 
There's a change to the `test` script that needs to be made.

**Review by sareh — APPROVED** [review]
👍 Looks great!

**Review by ChrisBAshton — COMMENTED** [review]
Trying to make it error. If I edit styledExampleButton.js:

* add a `margin-top` after a `margin` declaration
  * throws a `Unexpected shorthand "margin" after "margin-top"` as expected
* add a `made-up-property: foo` line
  * throws a `Unexpected unknown property "made-up-property"` as expected
* add a `!important`
  * doesn't complain. I think it probably should.
* add an additional `color` property
  * doesn't complain!
* add an additional `color` property, _alongside_ a `made-up-property`
  * throws the `Unexpected unknown property` _and_ `Unexpected duplicate "color"` errors

So, think I've found a bug in the stylelint...! 🤔 

1. Worth raising the bug with the right Stylelint repo?
2. Worth implementing our own `!important` rule? (overrideable with a comment, of course)

Looks good so far 👍

**Comment by dr3** [comment]
@ChrisBAshton I think that should be a separate issue as this is for moving the rules from simorgh. Happy to make a new issue and move if you agree

**Review by ChrisBAshton — APPROVED** [review]
Didn't realise we had any style linting in Simorgh! Have now fixed [this stale issue](https://github.com/BBC-News/simorgh/issues/564).

Have raised #39 and #40. Happy to approve this PR as-is. Thanks!

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**